scipy/stats/_hypotests_pythran.py

```diff
@@ -14,9 +14,10 @@ def _Dij(A, i, j):
     # See `somersd` References [2] bottom of page 309
     return A[i+1:, :j].sum() + A[:i, j+1:].sum()
 
-#pythran export _P(float[:,:])
-#pythran export _P(int[:,:])
-def _P(A):
+
+#pythran export _concordant_pairs(float[:,:])
```

Comment:
From the lint check, https://dev.azure.com/scipy-org/SciPy/_build/results?buildId=16514&view=logs&j=fdd89725-f657-5341-7d77-3c5c44c4f5b6&t=325fc03e-e0b9-5339-adaf-099a49a14600, we need to add one space now. 
`# pythran export _concordant_pairs(float[:,:])`
Also for line 19, 31, 32

Reply:
Done